cinema/models.py

```diff
@@ -78,9 +100,8 @@ class Ticket(models.Model):
     movie_session = models.ForeignKey(
         MovieSession, on_delete=models.CASCADE, related_name="tickets"
     )
-    order = models.ForeignKey(
-        Order, on_delete=models.CASCADE, related_name="tickets"
-    )
+    order = models.ForeignKey(Order, on_delete=models.CASCADE,
+                              related_name="tickets")
```

Comment:
we don't have problem with long line here, why changing it?
same in other files and this one in particular